packages/test/test-end-to-end-tests/src/test/summaryDataStoreStats.spec.ts.ts

```diff
@@ -94,13 +93,11 @@ describeNoCompat("Generate Summary Stats", (getTestObjectProvider) => {
         }
     }
 
-    const createContainer = async (): Promise<IContainer> => provider.createContainer(runtimeFactory);
-
-    let mockLogger: MockLogger;
+    const mockLogger = new MockLogger();
```

Comment:
You should create a new MockLogger for every test otherwise the logs from one test will mess up the subsequent tests.

Reply:
do i need to call createContainer for every test?

Reply:
wait...